packages/agent-utils/src/scripts/wrap-client/WrapClient.ts

```diff
@@ -361,6 +361,49 @@ export class WrapClient extends PolywrapClient {
           }
         },
       })))
+      .setPackage("plugin/cmd", PluginPackage.from(module => ({
+        "shellExec": async (params: { command: string, args?: string[] }) => {
+          logger.notice("CMD.EXEC = " + `${params.command} ${params.args}`);
+          return await workspace.shellExec(params.command, params.args);
+        },
+        "runPythonTest": async (params: { filename: string }) => {
```

Comment:
Why are we doing this here? Why not do this inside of the RunPythonTest function?